### PR TITLE
Quality: PowerShell Summit GH-3355

### LINKED ISSUE #3355: Quality: Get-VpnAuthProtocol
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples

### Article List

- add powershell to code blocks and separated output blocks.

### PR DESCRIPTION
**Pull request by XXLMandalorian013**
Add PS to code blocks and added out blocks that where tied into code blocks.

# PR Summary

- Resolves #3355

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/remoteaccess/Get-RemoteAccessRoutingDomain.md [hunk 1]
```diff
@@ -26,8 +26,11 @@ The **Get-RemoteAccessRoutingDomain** cmdlet retrieves routing domain configurat
 ## EXAMPLES
 
 ### Example 1: Retrieve configuration for a specific routing domain
+```powershell
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-RemoteAccessRoutingDomain.md:29]: We always want to have a blank line around block elements, like headings and code blocks

````suggestion

```powershell
````

File: docset/winserver2022-ps/remoteaccess/Get-RemoteAccessRoutingDomain.md [hunk 2]
```diff
@@ -26,8 +26,11 @@ The **Get-RemoteAccessRoutingDomain** cmdlet retrieves routing domain configurat
...
+Get-RemoteAccessRoutingDomain -Name Rd_01
 ```
-PS C:\> Get-RemoteAccessRoutingDomain -Name Rd_01
+
+```output
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-RemoteAccessRoutingDomain.md:33]:
````suggestion
```Output
````

File: docset/winserver2022-ps/remoteaccess/Get-RemoteAccessUserActivity.md [hunk 3]
```diff
@@ -52,24 +52,27 @@ However, only one of these filters can be used at a time.
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
-PS C:\>$startdate = Get-Date -Date "12/16/2011"
+```powershell
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-RemoteAccessUserActivity.md:55]:
````suggestion

```powershell
````

File: docset/winserver2022-ps/remoteaccess/Get-RemoteAccessUserActivity.md [hunk 4]
```diff
@@ -80,8 +83,11 @@ This example shows historic access details for a particular user.
 Query the accounting store for user activity details for User1 between the start date and end date.
 
 ### EXAMPLE 2
+```powershell
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-RemoteAccessUserActivity.md:86]:
````suggestion

```powershell
````

File: docset/winserver2022-ps/remoteaccess/Get-RoutingProtocolPreference.md [hunk 5]
```diff
@@ -29,8 +29,11 @@ Use the Set-RoutingProtocolPreference cmdlet to modify preferences.
 ## EXAMPLES
 
 ### Example 1: Display preferences
+```powershell
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-RoutingProtocolPreference.md:32]:
````suggestion

```powershell
````

File: docset/winserver2022-ps/remoteaccess/Get-VpnAuthProtocol.md [hunk 6]
```diff
@@ -25,8 +25,11 @@ The **Get-VpnAuthProtocol** cmdlet retrieves VPN authentication parameters for i
 ## EXAMPLES
 
 ### EXAMPLE 1
+```powershell
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/remoteaccess/Get-VpnAuthProtocol.md:28]:
````suggestion

```powershell
````

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]

**Comment by sdwheeler** [comment]
Retrigger build

**Pull request merged**